### PR TITLE
[Merged by Bors] - fix: shorten lines with 101 characters

### PR DESCRIPTION
**Pull request by adomani**
Found by the long line syntax linter at #15097.

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/Computability/AkraBazzi/AkraBazzi.lean [hunk 1]
```diff
@@ -464,7 +464,7 @@ lemma isEquivalent_smoothingFn_sub_self (i : α) :
             have h_log_pos : 0 < log n := Real.log_pos <| by aesop
             simp only [one_div]
             rw [inv_sub_inv (by have := hn' i; positivity) (by aesop)]
-      _ =ᶠ[atTop] (fun (n : ℕ) => (log n - log (b i) - log n) / ((log (b i) + log n) * log n))  := by
+      _ =ᶠ[atTop] (fun (n : ℕ) ↦ (log n - log (b i) - log n) / ((log (b i) + log n) * log n))  := by
```
Comment (grunweg) [inline Mathlib/Computability/AkraBazzi/AkraBazzi.lean:467]: Here's another character saved for you :-)
```suggestion
      _ =ᶠ[atTop] (fun (n : ℕ) ↦ (log n - log (b i) - log n) / ((log (b i) + log n) * log n)) := by

**Review by grunweg — COMMENTED** [review]
All of these look like reasonable choices to me - thanks for doing this clean-up.
maintainer merge

**Comment by riccardobrasca** [comment]
Thanks 🎉

If CI passes, please remove the label `awaiting-CI` and merge this yourself, by adding a comment `bors r+`.

bors d+

File: Mathlib/Computability/AkraBazzi/AkraBazzi.lean (hunk 1 above)
Comment (adomani) [reply Mathlib/Computability/AkraBazzi/AkraBazzi.lean:467]: Yes, I had missed this when I merged the PR.  I already opened a PR with this (and a couple more) changes: #15107.

**Pull request closed without merging**